src/call.rs

```diff
@@ -53,7 +53,7 @@ impl Call {
 
 impl SizedBytes for Call {
     fn serialized_size(&self) -> usize {
-        ContractId::LEN + 2 * WORD_SIZE
+        WORD_SIZE.saturating_mul(2).saturating_add(ContractId::LEN)
```

Comment:
I think the risk is fairly low here, since these values are all known at compile time and are constant. These constants translate into `8 * 2 + 32`, which seems a little overkill to add checked math error handling around.